tests/unit/service/test_static_source_table.py

```diff
@@ -0,0 +1,129 @@
+"""
+Test for StaticSourceTableService
+"""
+import textwrap
+from unittest.mock import AsyncMock, Mock
+
+import pandas as pd
+import pytest
+
+from featurebyte.enum import SourceType
+from featurebyte.models.request_input import SourceTableRequestInput
+from featurebyte.models.static_source_table import StaticSourceTableModel
+from featurebyte.query_graph.model.common_table import TabularSource
+from featurebyte.query_graph.node.schema import TableDetails
+from featurebyte.session.base import BaseSession
+
+
+@pytest.fixture(name="static_source_table_from_source_table")
+def static_source_table_from_source_table_fixture(event_table):
+    """
+    Fixture for an StaticSourceTable from a source table
+    """
+    request_input = SourceTableRequestInput(source=event_table.tabular_source)
+    location = TabularSource(
+        **{
+            "feature_store_id": event_table.tabular_source.feature_store_id,
+            "table_details": {
+                "database_name": "fb_database",
+                "schema_name": "fb_schema",
+                "table_name": "fb_materialized_table",
+            },
+        }
+    )
+    return StaticSourceTableModel(
+        name="static_source_table_from_source_table",
+        location=location,
+        request_input=request_input,
+        columns_info=[
+            {"name": "a", "dtype": "INT"},
+            {"name": "b", "dtype": "INT"},
+            {"name": "c", "dtype": "INT"},
+        ],
+        num_rows=1000,
+        most_recent_point_in_time="2023-01-15T10:00:00",
+    )
+
+
+@pytest.fixture(name="table_details")
```

Comment:
can we reuse the fixture in `test_observation_table` for this too?

Reply:
I realized that some tests are unnecessary since the service shares the same based class with several other classes that already provided test coverage. Will be removing these tests, and as a result these fixtures are no longer required.